contracts/zero-ex/contracts/src/transformers/FillQuoteTransformer.sol

```diff
@@ -178,9 +182,14 @@ contract FillQuoteTransformer is
         // Approve the ERC20 proxy to spend `sellToken`.
         data.sellToken.approveIfBelow(erc20Proxy, data.fillAmount);
 
-        // Fill the orders.
         state.protocolFee = exchange.protocolFeeMultiplier().safeMul(tx.gasprice);
         state.ethRemaining = address(this).balance;
+        // RFQT orders can only be filled if we have a valid calldata hash
+        // (calldata was signed), and the actual taker matches the RFQT taker (if set).
+        state.isRfqtAllowed = context.callDataHash != bytes32(0)
+            && (data.rfqtTakerAddress == address(0) || context.taker == data.rfqtTakerAddress);
```

Comment:
This buys us time to work on properly propagating the `rfqtTakerAddress` from 0x-API down to AS. We will just set `data.rfqtTakerAddress = 0` for the moment.